### ISSUE TITLE
ibm-search  AVT issue

### ISSUE THREAD

**Issue by aaratichavan** [IBM/carbon-components-angular#1825]
After using two ibm-search in one page then facing accessibility issue
ibm-search accessibility violation issue - Multiple elements with "search" role do not have unique labels

![image](https://user-images.githubusercontent.com/69956330/118820975-6874e180-b8d4-11eb-8701-33fe2846394c.png)

**Comment by cal-smith** [comment]
This is an application level issue. Please use the `ariaLabel` input to specify a label for each search on the page. There is nothing we can do to fix this at the library level.
